src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { ChangeEventHandler, FormEvent, useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [text, setText] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [todos, setTodos] = useState(
+    todosFromServer.map(todo => ({
+      ...todo,
+      user: usersFromServer.find(user => user.id === todo.userId) || null,
+    })),
```

Comment:
nice, but make for this separete func

Reply:
Did you mean this? 

`  const users = todosFromServer.map(
    todo => usersFromServer.find(user => user.id === todo.userId) || null,
  );
  const [todos, setTodos] = useState(
    todosFromServer.map(todo => ({
      ...todo,
      user: users,
    })),
  );`